src/Dash/Router/RouteCollection/RouteCollection.php

```diff
@@ -101,35 +102,15 @@ public function get($name)
         return $route;
     }
 
-    public function current()
-    {
-        $node = current($this->routes);
-        return ($node !== false ? $this->get(key($this->routes)) : false);
-    }
-
-    public function key()
-    {
-        return key($this->routes);
-    }
-
-    public function next()
-    {
-        $node = next($this->routes);
-        return ($node !== false ? $this->get(key($this->routes)) : false);
-    }
-
-    public function rewind()
+    public function getIterator()
```

Comment:
Maybe we could make the route collection NOT implemeting IteratorAggregate, and instead rename "getIterator" to "getRoutes".